components/globals/ErrorPanel.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import { useTranslation } from "next-i18next";
+import { useSession } from "next-auth/react";
+
+import { LinkButton } from "@components/globals";
+import { BackArrowIcon } from "@components/form-builder/icons";
+
+export const ErrorPanel = ({
+  title,
+  children,
+  headingTag: HeadingTag = "h2",
+}: {
+  title?: string;
+  children?: JSX.Element | string;
+  headingTag?: "h1" | "h2";
+}) => {
+  const { t, i18n } = useTranslation("common");
+  const { status } = useSession();
+
+  // default content
+  const defaultTitle = t("errorPanel.defaultTitle");
+  const defaultMessage = <p>{t("errorPanel.defaultMessage")}</p>;
+
+  // links
+  const homeHref =
+    status === "authenticated"
+      ? `/${i18n.language}/form-builder/support`
+      : `/${i18n.language}/form-builder`;
+
+  const homeText =
+    status === "authenticated" ? t("errorPanel.cta.yourForms") : t("errorPanel.cta.home");
+
+  const supportHref = `/${i18n.language}/form-builder/support`;
```

Comment:
It looks like both links will be to support for an authenticated user. Maybe this is intentional?

Reply:
Not intentional -- will fix ... messed up with one of my re-factors.